rsts/deployment/cluster_config/general.rst

```diff
@@ -103,21 +101,21 @@ Then run the below command to ensure that when the admin fills in cluster resour
 Refer to the :ref:`docs <flytectl:flytectl_update_cluster-resource-attribute>` to learn more about the command and its supported flag(s).
 
 
-To fetch and verify the individual project-domain attributes, similarly you can run:
+To fetch and verify the individual project-domain attributes:
 
 .. prompt:: bash
 
     flytectl get cluster-resource-attribute -p flyteexamples -d development
 
 Refer to the :ref:`docs <flytectl:flytectl_get_task-resource-attribute>` to learn more about the command and its supported flag(s).
 
-The above-updated values will, in turn, be used to fill in the template fields for the flyteexamples-development namespace.
+You can use these updated values to fill the template fields for the flyteexamples-development namespace.
```

Comment:
A user doesn't fill in the template fields; it's done by Flyte. I think it's ok to use passive voice in these scenarios.

Reply:
"Flyte uses these updated values to fill the template fields for the flyteexamples-development namespace." sounds better IMO.